geode-core/src/integrationTest/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImplJUnitTest.java

```diff
@@ -411,6 +411,53 @@ public void testLocatorUpdateIntervalZero() {
     assertEquals(updateLocatorInterval, source.getLocatorUpdateInterval());
   }
 
+
+  @Test
+  public void testLocatorIpChange2() {
```

Comment:
Would you be able to modify the test name to describe what we are testing ?  I don't know much about the code and unable to quickly decipher what we are trying to test.

Reply:
I agree it would be good to have a more descriptive name here